test/e2e/webhook_test.go

```diff
@@ -1291,3 +1293,68 @@ var _ = Describe("webhook tests for ResourceOverride UPDATE operations", Ordered
 		}, testutils.PollTimeout, testutils.PollInterval).Should(Succeed())
 	})
 })
+
+var _ = Describe("webhook tests for operations on ARM managed resources", Ordered, func() {
+	BeforeAll(func() {
+		By("creating a managed namespace")
+		createManagedNamespace()
+	})
+
+	AfterAll(func() {
+		By("deleting the managed namespace")
+		ns := managedNamespace()
+		Expect(hubClient.Delete(ctx, &ns)).Should(SatisfyAny(Succeed(), utils.NotFoundMatcher{}), "Failed to delete the managed namespace")
+		Eventually(func() error {
+			if err := hubClient.Get(ctx, types.NamespacedName{Name: ns.Name}, &corev1.Namespace{}); !k8sErrors.IsNotFound(err) {
+				return fmt.Errorf("The managed namespace still exists or an unexpected error occurred: %w", err)
+			}
+			return nil
+		}, workloadEventuallyDuration, eventuallyInterval).Should(Succeed(), "Failed to remove the managed namespace %s", ns.Name)
```

Comment:
nit: to keep in consistent, let's use testutils.PollTimeout, testutils.PollInterval